staging/src/k8s.io/apimachinery/pkg/api/validation/objectmeta.go

```diff
@@ -176,7 +176,9 @@ func ValidateObjectMetaAccessor(meta metav1.Object, requiresNamespace bool, name
 			allErrs = append(allErrs, field.Invalid(fldPath.Child("clusterName"), meta.GetClusterName(), msg))
 		}
 	}
-	for _, entry := range meta.GetManagedFields() {
+	flds := meta.GetManagedFields()
+	for i := range flds {
+		entry := flds[i]
```

Comment:
I don't think this change is really interesting overall, but sure why not.

`flds` is not a great name, maybe just `fields`?
Drop the temporary `entry` here and do `fields[i].Manager` below.

Same for the other hunk.